apps/web/src/app/[locale]/challenge/_components/comments/comment-input.tsx

```diff
@@ -21,6 +22,7 @@ export function CommentInput({ mode, onCancel, onChange, value, placeholder, onS
   const { toast } = useToast();
   const [commentMode, setCommentMode] = useState<'editor' | 'preview'>('editor');
   const [isSubmitting, setIsSubmitting] = useState(false);
+  const [submittable, setSubmittable] = useState(true);
```

Comment:
can't we just leverage zod here and change this into a form

Reply:
probably a good idea but I haven't worked with zod before. I'll try to make it work